src/index.css

```diff
@@ -9,6 +9,8 @@ body {
   background-color:darkslategrey;
   padding:0;
   margin:0;
+  width: 100vw;
+  height: 100vh;
```

Comment:
Here's the CSS that says "use 100% of the viewport width and height".

Reply:
That's perfect, it gets rid of the white border around the edge from a rounding error I was using.